ec2/spark_ec2.py

```diff
@@ -569,15 +569,34 @@ def launch_cluster(conn, opts, cluster_name):
         master_nodes = master_res.instances
         print "Launched master in %s, regid = %s" % (zone, master_res.id)
 
```

Comment:
Yeah, It is true. I will make a new commit as soon as possible. 
I think I will add 5 seconds sleep time before tagging master and make some modification in the comment.